v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -47,8 +50,14 @@ def raw_dataset_path(self, reference_genome: ReferenceGenome) -> str | list[str]
         return CONFIG[self][reference_genome][RAW_DATASET_PATH]
 
     def get_ht(self, reference_genome: ReferenceGenome) -> hl.Table:
+        file_name = (
```

Comment:
ooof, this is annoying... I wasn't able to come up with something better in the few min I've been thinking about this though.

Reply:
what if we make this explicitly configured in the CONFIG with like a MODULE_NAME key, so then here we would just have 
```
module_file_name = CONFIG[self].get(MODULE_NAME, self.name)
```